client/app/scripts/liveblog-edit/views/settings.html

```diff
@@ -69,41 +69,63 @@ <h2 translate>General</h2>
                 <!-- Users -->
                 <div class="tab-content outer" ng-if="!settings.generalTab">
                     <div class="header">
-                        <h2 translate>Users</h2>
+                        <h2 translate>Team</h2>
                     </div>
                      <div class="content">
-                        <form class="form-horizontal" name="blog-creator">
-                            <div class="form-group">
-                                <label for="inputLanguage" class="control-label">Image</label>
-                                <div class="form-input profile-pic">
-                                     <figure class="avatar">
-                                        <img sd-user-avatar data-src="settings.original_creator.user.picture_url">
-                                    </figure>
-                                </div>
-                            </div>
-                            <div class="form-group">
-                                <label for="inputTheme" class="control-label">Full Name</label>
-                                <div class="form-input" data="original-creator-display-name">
-                                   {{ settings.original_creator.display_name }}
-                                </div>
+                        <div class="settings-subsection">
+                            <div class="subsettings-header">
+                                <h3 translate>Owner</h3>
                             </div>
-                            <div class="form-group">
-                                <label for="inputEmbedCode" class="control-label">Username</label>
-                                <div class="form-input" data="original-creator-username">
-                                    {{ settings.original_creator.username }}
+                            <div class="subsettings-content">
+                                <div class="inline-content first-block">
+                                    <div class="inline-content padded-right">
+                                        <div class="form-input profile-pic">
+                                             <figure class="avatar">
+                                                <img data-src="settings.original_creator.user.picture_url">
+                                            </figure>
+                                        </div>
+                                    </div>
+                                    <div class="inline-content">
+                                        <div class="form-input" data="original-creator-display-name">
+                                           <strong>{{ settings.original_creator.display_name }}</strong>
+                                        </div>
+                                        <div class="form-input" data="original-creator-username">
+                                            {{ settings.original_creator.username }}
```

Comment:
Does the original creator change? If not, one-time binding can be used here (and the same for the display_name a few lines above).